nipype/interfaces/dcm2nii.py

```diff
@@ -360,11 +360,6 @@ class Dcm2niix(CommandLine):
     >>> converter.inputs.output_dir = 'ds005'
     >>> converter.cmdline
     'dcm2niix -b y -z y -5 -x n -t n -m n -o ds005 -s n -v n dicomdir'
-
-    >>> flags = '-'.join([val.strip() + ' ' for val in sorted(' '.join(converter.cmdline.split()[1:-1]).split('-'))])
-    >>> flags
-    ' -5 -b y -m n -o ds005 -s n -t n -v n -x n -z y '
-    >>> converter.run() # doctest: +SKIP
```

Comment:
I think we did still want the `converter.run()` line, though.

Reply:
:sweat_smile: